onnxruntime/contrib_ops/cuda/transformers/generation_device_helper.cc

```diff
@@ -27,6 +27,20 @@ namespace onnxruntime {
 namespace contrib {
 namespace GenerationCudaDeviceHelper {
 
+size_t GetTensorSizeInBytes(const Tensor& tensor) {
```

Comment:
FYI, there is Tensor::SizeInBytes()
https://github.com/microsoft/onnxruntime/blob/fb85b31facb9fb3fc99c76f99c93ea8f06ada39b/onnxruntime/core/framework/tensor.cc#L89

Reply:
Thanks!